newIDE/app/src/Profile/EditProfileDialog.js

```diff
@@ -36,12 +43,20 @@ import Text from '../UI/Text';
 import TextButton from '../UI/TextButton';
 import useAlertDialog from '../UI/Alert/useAlertDialog';
 import Form from '../UI/Form';
+import RaisedButton from '../UI/RaisedButton';
+import Coin from '../Credits/Icons/Coin';
+import { sendGitHubStarUpdated } from '../Utils/Analytics/EventSender';
 
-type Props = {|
+export type Props = {|
```

Comment:
Could you name the Props EditProfileDialogProps in order to avoid having VSCode suggest `Props` everywhere?

Reply:
Yep!